terminus-ui/tabs/src/body/tab-body.component.spec.ts

```diff
@@ -0,0 +1,119 @@
+import { PortalModule } from '@angular/cdk/portal';
+import { CommonModule } from '@angular/common';
+import {
+  async,
+  ComponentFixture,
+  TestBed,
+} from '@angular/core/testing';
+import { MatRippleModule } from '@angular/material';
+import { By } from '@angular/platform-browser';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import {
+  TsTabBodyComponent,
+  TsTabBodyHostDirective,
+  TsTabsModule,
+} from '@terminus/ui/tabs';
+import * as testComponents from '@terminus/ui/tabs/testing';
+
+
+
+
+describe(`TsTabBodyComponent`, function() {
+  let fixture: ComponentFixture<testComponents.TabBody>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        CommonModule,
+        PortalModule,
+        MatRippleModule,
+        NoopAnimationsModule,
+        TsTabsModule,
+      ],
+      declarations: [
+        testComponents.TabBody,
+      ],
+    });
+
+    TestBed.compileComponents();
+  }));
+
+
+  describe(`when initialized as center`, () => {
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(testComponents.TabBody);
+    });
+
+
+    test(`should be center position if origin is unchanged`, () => {
+      fixture.detectChanges();
+      fixture.componentInstance.position = 0;
+      fixture.detectChanges();
+
+      expect(fixture.componentInstance.tabBody.positionState).toEqual('center');
```

Comment:
DOM?

Reply:
This sets up the animations to move tabs. So we're testing the DOM part when using our helper `checkSelectedIndex` and this is just further testing the methods themselves.